src/installer.ts

```diff
@@ -3,11 +3,11 @@ let tempDirectory = process.env['RUNNER_TEMP'] || '';
 import * as core from '@actions/core';
 import * as io from '@actions/io';
 import * as exec from '@actions/exec';
+import * as httpm from '@actions/http-client';
```

Comment:
consider add retries and error checking (or switch to getJson which does error checking)